README.md

```diff
@@ -322,10 +322,11 @@ intereceptor.on('connection', ({ client }) => {
 
 The `connection` event exposes the following arguments:
 
-| Name     | Type                                                      | Description                                                      |
-| -------- | --------------------------------------------------------- | ---------------------------------------------------------------- |
-| `client` | [`WebSocketClientConnection`](#websocketclientconnection) | An object representing a connected WebSocket client instance.    |
-| `server` | [`WebSocketServerConnection`](#websocketserverconnection) | An object representing the original WebSocket server connection. |
+| Name        | Type                                                      | Description                                                      |
+| ----------- | --------------------------------------------------------- | ---------------------------------------------------------------- |
+| `client`    | [`WebSocketClientConnection`](#websocketclientconnection) | An object representing a connected WebSocket client instance.    |
+| `server`    | [`WebSocketServerConnection`](#websocketserverconnection) | An object representing the original WebSocket server connection. |
+| `protocols` | <code>string &#124; string[] &#124; undefined</code>      | The protocols supported by the WebSocket client.                 |
```

Comment:
Maybe something like `info` would be more long-lasting? I don't want to prematurely optimize here but I'm thinking just in case we need to expose additional connection info in the future, this may be a safer API to extend. 

Wdyt?

Reply:
`client.protocols`? 🤔

Reply:
Again, all good with me. I don't have strong opinions on how to expose those values 😉